pages/_app.js

```diff
@@ -1,21 +1,36 @@
 import React from 'react';
 import Head from 'next/head';
+import { useState, useEffect } from 'react'
+import Pool from '../lib/webrtc/Pool'
 
 import { ThemeProvider, CSSReset, ColorModeProvider } from '@chakra-ui/core';
 
-import theme from './_theme';
-
-const App = ({ Component }) => (
-    <ThemeProvider theme={theme}>
-        <CSSReset />
-        <ColorModeProvider>
-            <Head>
-                <title>Oeaki</title>
-                <link rel="shortcut icon" href="/favicon.ico" />
-            </Head>
-            <Component />
-        </ColorModeProvider>
-    </ThemeProvider>
-);
+import theme from '../lib/theme';
+
+
+const App = ({ Component }) => {
+    const [pool, setPool] = useState(null);
+
+    useEffect(() => {
+        if (typeof window !== 'undefined') {
+            window.pool = new Pool(`${window.location.protocol === 'http:' ? 'ws' : 'wss'}://${window.location.hostname}:3000`)
+            setPool(pool);
+        }
+    }, []);
```

Comment:
I don't think we want to init WebRTC connexion here. It would mean that all users on the homepage would create a WebRTC connexion.
As I see it, the pool should be init when the user is in the game lobby. So we can create a "room" server side identified by the party id where all users can share WebRTC informations.

Reply:
Actually, we could set it here with the `useEffect` watching an id in the url queries :
```js
useEffect(() => { if(query.partyId) { ... } , [query.partyId]);
```